Dockerfile

```
@@ -1,3 +1,9 @@
-FROM registry.gitlab.com/pages/hugo/hugo_extended:0.101.0
-CMD [ "serve", "-D", "--bind", "0.0.0.0" ]
-ENTRYPOINT [ "hugo" ]
+FROM registry.gitlab.com/pages/hugo/hugo_extended:0.101.0 as builder
+RUN mkdir -p /site
+WORKDIR /site
+COPY . /site
+RUN hugo
+
+FROM nginx
```

IIRC [you'd mentioned](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/249#discussion_r940733041) you like the alpine images, is there a reason you're not using that here?

```suggestion
FROM nginx:alpine
```